src/components/dashboard/Claim.js

```diff
@@ -253,6 +253,8 @@ const Claim = props => {
     log.debug('from FR:', { isValid, screenProps })
     try {
       if (isValid && (await goodWallet.isCitizen())) {
+        //collect invite bonus
+        goodWallet.collectInviteBounty()
```

Comment:
```suggestion
        await goodWallet.collectInviteBounty()
```

probably we should await before sending claim request ? to make sure bonuses will be applied ?

Reply:
no i want this to happen in background